plugins/GSdx/res/tfx.fx

```diff
@@ -52,11 +52,12 @@
 #endif
 
 #define SW_BLEND (PS_BLEND_A || PS_BLEND_B || PS_BLEND_D)
+#define PS_AEM_FMT (PS_FMT & 3)
 
 struct VS_INPUT
 {
 	float2 st : TEXCOORD0;
-	float4 c : COLOR0;
+	uint4 c : COLOR0;
```

Comment:
Don't we need to change it across all shaders?

Reply:
I don't think so. The cpp code change is for `"tfx.fx", "vs_main"` so other shaders should be unaffected. I would prefer not to mess with those.